sources/packages/api/test/users.e2e-spec.ts

```diff
@@ -0,0 +1,62 @@
+require("../env_setup");
```

Comment:
Should we have anotherfolder under test to segragate tests for institution users and students?

Reply:
This method is not institution-specific. It is under the same user endpoint already in use for students.
We can discuss how to organize on folder based on the endpoint, maybe.